### ISSUE TITLE
Doesn't deauth 

### ISSUE THREAD

**Issue by Appeng71** [wfhstudio/WiFi-Stealer#4]
Everything works fine but when i press the button for deauth the router , it doesn't work. only clone ssid showing and password grabbing works fine. How to solve deauth issue. Should i need to modify  any thing?

**Comment by wfhstudio** [comment]
You need to get board esp8266 deauther and put in your preference .. see in
spacehuh github

**Comment by Appeng71** [comment]
oh! got it.... basically i have to deauth wifi with another esp8266...is it correct?.... i thought that  deauth and captive portal works same time.... by the way you are doing great job.... but if you start in English language , your yt channel will go viral

**Comment by wfhstudio** [comment]
No when you flash your esp8266 use this board

**Comment by Appeng71** [comment]
ok got....let me try again...i will update you

**Comment by Appeng71** [comment]
nice bro now its working perfectly ....thanks

**Comment by wfhstudio** [comment]
Glad to hear that ... Enjoy
